src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx

```diff
@@ -16,63 +16,49 @@ import collaboration_discussion_board_new from 'images/collaboration-discussion-
 
-You may notice the **Share** menu disabled on some pages. By design, in a multi-account organization you can't discuss pages that either don't have data associated with a specific account or pages that present data belonging to multiple accounts in a single view. If you select a specific account using the account picker or click through to a specific item or entity you will usually see the **Share** menu.
+You may notice the **Share** menu disabled on some pages. By design, in a multi-account organization, you can't discuss pages that don't have data associated with a specific account or pages that present data belonging to multiple accounts in a single view. If you select a specific account using the account picker or click through to a particular item or entity, you will usually see the **Share** menu.
 
 <img
-  title="Comment on any page "
-  alt="A screenshot of the UI that depicts commenting on a specific page."
+  title= "Comment on any page"
+  alt= "A screenshot of the UI that depicts commenting on a specific page."
   src={collaborationCommentonAnyPage}
-  style={{ maxWidth: '60%' }}
+  style={{ maxWidth: '60%'}}
 />
 
-Pages that have existing discussions will also have a new **Discussions** button in the top navigation that will indicate how many different discussion threads exist on the page. If you click **Discussions** a comment panel will open where you can view and contribute to those discussions. If any of the discussions are in **Open** status the button will be green as depicted in the screenshot above. The green highlighting calls attention to unanswered questions or unresolved issues you may be able to contribute to. 
+Pages with existing discussions will also have a new **Discussions** button in the top navigation that indicates how many different discussion threads exist on the page. If you click **Discussions**, a comment panel will open where you can view and contribute to those discussions. If any discussions are in **Open** status, the button will be green, as depicted in the screenshot above. The green highlighting calls attention to unanswered questions or unresolved issues you can contribute to. 
 
 ## At-mention your teammates and notify by email [#at-mention-your-teammates]
 
 Like other collaboration tools, when you at-mention (@) your teammates in a comment, they'll receive an email notification. 
 
 ## Built-in screenshot sharing [#built-in-screenshots]
 
-Attach screenshots to your comments by clicking our integrated screenshot selection tool. Just click the “screenshot” icon, select the portion of the screen you want, and submit your comment. 
+Attach screenshots to your comments by clicking our integrated screenshot selection tool. Just click the "screenshot" icon, select the portion of the screen you want, and submit your comment. 
```

Comment:
@homelessbirds I'm not wild about "screenshot" icon. Could you either remove the quotation marks or insert an inline image showing what that is? Will this be obvious to users?

Reply:
I fixed this. It is obvious to users, so I just removed the quotation marks.